### PR TITLE
Added verdict storage

### PR DESCRIPTION
**Pull request by quoll**
issue-277 on CTIA needs to store verdicts that get created in response to judgements. This adds some functionality to allow for that.

### CODE REVIEW

File: src/ctim/schemas/verdict.clj [hunk 1]
```diff
@@ -11,7 +16,32 @@ the active verdict.  If there is more than one Judgement with that
...
+    login :- s/Str]
+   (realize-verdict new-verdict (make-id) login))
+  ([new-verdict :- Verdict
+    id :- s/Str
+    login :- s/Str]
+   (let [now (time/now)]
+     (assoc new-verdict
+            :id id
+            :owner login
+            :created now))))
```
Comment (yogsototh) [inline src/ctim/schemas/verdict.clj]: The `make-id` and `realised-verdict` should be present only in CTIA not in CTIA. So you could remove them.

Comment (quoll) [inline src/ctim/schemas/verdict.clj]: I realized this later in the day, and did it. Somehow I've messed up git, and I've ended up with a new PR that does this.

**Comment by polygloton** [comment]
Related to threatgrid/ctia#277

**Comment by polygloton** [comment]
Replaced by #12

**Pull request closed without merging**